data/json/items/containers.json

```diff
@@ -128,6 +128,23 @@
     "color": "light_gray",
     "contains": 24,
     "flags": [ "TRADER_AVOID" ]
+	},
+	{
+    "id": "bag_zipper",
+    "type": "CONTAINER",
+    "category": "other",
+    "name": "zipper bag",
+    "description": "An inexpensive flexible rectangular storage bag on its typical small size.  Transparent and made of plastic, it can be sealed and opened by a slider which works in a similar way to a zip fastener.",
+    "weight": 2,
+    "volume": 0,
+    "price": 0,
+	"rigid": false,
+    "material": "plastic",
+    "symbol": ")",
+    "color": "light_gray",
+    "contains": 1,
+	"seals": true,
+	"flags": [ "TRADER_AVOID" ]
```

Comment:
It's not required, but adding a `looks_like` field would be neat. Perhaps `"looks_like": "bag_plastic"`?

Reply:
Sure, done. I kinda expected what `looks_like` would do but I wasn't sure and I didn't want to ruin anything so decided to not add. Either way, thank you!